core_backend/app/__init__.py

```diff
@@ -21,6 +21,8 @@ def create_app() -> FastAPI:
         "https://localhost:3000",
         "http://localhost",
         "http://localhost:3000",
+        f"http://{DOMAIN}",
+        f"http://{DOMAIN}:3000",
```

Comment:
This is a workaround for the following error:
![image](https://github.com/IDinsight/aaq-core/assets/7042047/d64a89f1-34f4-4a6b-a4ad-af85fc07742c)